eZ/Publish/Core/Repository/ContentTypeService.php

```diff
@@ -168,35 +168,23 @@ public function createContentTypeGroup(ContentTypeGroupCreateStruct  $contentTyp
     }
 
     /**
-     * Get a Content Type Group object by id.
-     *
-     * @throws \eZ\Publish\API\Repository\Exceptions\NotFoundException If group can not be found
-     *
-     * @param mixed $contentTypeGroupId
-     *
-     * @return \eZ\Publish\API\Repository\Values\ContentType\ContentTypeGroup
+     * {@inheritdoc}.
```

Comment:
A small nitpick on what I've just discovered. This dot after `{@inheritdoc}` is not needed (and php-cs-fixer is ok with that). When present it becomes a part of doc in the new line. Looks bad in PhpStorm. 
*Applies to several places in the PR.*
![inheritdoc_dot](https://user-images.githubusercontent.com/7099219/26881998-62d289be-4b99-11e7-8e1c-48870cc5025d.png)